packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -63,11 +65,29 @@ export default class SaytDriverPlugin implements Plugin {
     fields: ['*'],
   }
 
-  constructor() {
+  /**
+   * Default product transformer identity function.
+   * Intended to be overwritten by passing a custom product transformer.
+   *
+   * @param product The product to be returned as-is.
+   * @returns The received product object.
+   */
+  transformProduct: ProductTransformer<P> = ((product: Record): Record => product) as any;
+
+  /**
+   * Binds relevant functions. Sets transformProduct property if a
+   * product transformer function is passed.
+   */
+  constructor(options: SaytDriverOptions = {}) {
```

Comment:
This change has a corresponding suggestion where the interface is defined.

```suggestion
  constructor(options: SaytDriverOptions<P> = {}) {
```